src/Plugins/Artemis.Plugins.Modules.General/DataModels/Windows/WindowDataModel.cs

```diff
@@ -14,6 +20,28 @@ public WindowDataModel(Process process)
 
             // Accessing MainModule requires admin privileges, this way does not
             ProgramLocation = process.GetProcessFilename();
+
+            // Get Icon colors
+            if(File.Exists(ProgramLocation)) 
+            {
+                MemoryStream mem = new MemoryStream();
+                Icon.ExtractAssociatedIcon(ProgramLocation).Save(mem);
+                mem.Seek(0, SeekOrigin.Begin);
+                var skbm = SKBitmap.Decode(mem);
+                mem.Close();
+
+                IColorQuantizerService colorQuantizer = new ColorQuantizerService();
+                var skClrs = colorQuantizer.Quantize(skbm.Pixels.ToList(), 256).ToList();
```

Comment:
We're no longer using `var` but I don't mind reformatting code after merging, just a sidenote.